forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useRef, useEffect } from "react";
+import Modal from "react-bootstrap/Modal";
+import { CustomButton } from "./Button";
+import { FormTextArea } from "./FormTextArea";
+import { CloseIcon } from "../SvgIcons/index";
+import { FormInput } from "../CustomComponents/FormInput";
```

Comment:
replace with "./FormInput" instead of "../CustomComponents/FormInput"

Reply:
Solved